Token.sol

```diff
@@ -34,6 +34,12 @@ contract TokenInterface {
     mapping (address => uint256) balances;
     mapping (address => mapping (address => uint256)) allowed;
 
+    /// Public variables of the token, all used for display 
```

Comment:
Yes, I don't get this either. Where is the constructor of your functions? Initial supply is not set on this page either